R/model-tools.R

```diff
@@ -371,3 +371,92 @@ enw_model <- function(model = system.file(
   }
   return(model)
 }
+
+#' Expose `epinowcast` stan functions in R
+#'
+#' @description This function facilitates the exposure of Stan functions from
+#' the [epinowcast]() package in R. It utilizes the [expose_functions()] method
+#' of [cmdstanr::CmdStanModel] or this purpose. This function is useful for
+#' developers and contributors to the [epinowcast] package, as well as for
+#' users interested in exploring and prototyping with model functionalities.
+#'
+#' @param files A character vector specifying the names of Stan files to be
+#' exposed. These must be in the `include` directory. Defaults to all Stan
+#' files in the `include` directory. Note that the following files are
+#' overloaded and cannot be exposed: "delay_lpmf.stan",
```

Comment:
```suggestion
#' files in the `include` directory. Note that the following files contain
#' overloaded functions and cannot be exposed: "delay_lpmf.stan",
```

Reply:
Also, is "delay_lpmf.stan" really a problem?

Reply:
I think so as it depends on functions that can't be exposed.